.stylelintrc.js

```diff
@@ -14,5 +14,8 @@ module.exports = {
     'carbon/motion-easing-use': true,
     'carbon/theme-use': true,
     'carbon/type-use': true,
+
+    // CSS Logical properties
+    'csstools/use-logical': 'ignore',
```

Comment:
We added the rule but for now, we are ignoring so that we can roll them out in separate PR's, the last step would be to enable it at the end

Reply:
Just noticed we already have this listed in `.stylelintrc.js`, don't need to add it again